doc/modules/clustering.rst

```diff
@@ -1444,6 +1444,10 @@ will not necessarily be close to zero.::
   * `Wikipedia entry for the Rand index
     <https://en.wikipedia.org/wiki/Rand_index#Adjusted_Rand_index>`_
 
+  * `Minimum adjusted Rand index for two clusterings of a given size
```

Comment:
Thanks for the PR! 

This would be nice if you could use a more similar link as in https://github.com/scikit-learn/scikit-learn/pull/24019 so I guess something like:
```rst
:doi:`Minimum adjusted Rand index for two clusterings of a given size, 2022, J. E. Chacón and A. I. Rastrojo <10.1007/s11634-022-00491-w>`
```

Ideally double-check the HTML rendering by generating the doc (have a look at the contributing doc) or wait for the CI to complete and click on the "check rendered doc" CI status

Reply:
Hey, thanks for the comments. I added the `:doi` flag as you requested and checked that it renders fine locally. I will check the CI job as well once it is done.